src/EditorFeatures/Core/Shared/Preview/PreviewWorkspace.cs

```diff
@@ -60,6 +60,11 @@ public override void OpenDocument(DocumentId documentId, bool activate = true)
             this.OnDocumentOpened(documentId, text.Container);
         }
 
+        /// <summary>
+        /// Puts the specified additional document into the open state.
+        /// </summary>
+        /// <param name="documentId">The <see cref="DocumentId"/> to open.</param>
+        /// <param name="activate">Ignored - not necessary for additional documents.</param>
```

Comment:
Isn't the statement for the param here more "this doesn't matter in the preview workspace?"